src/transformers/feature_extraction_utils.py

```diff
@@ -537,10 +538,15 @@ def get_feature_extractor_dict(
                 f"loading configuration file {feature_extractor_file} from cache at {resolved_feature_extractor_file}"
             )
 
-        if "auto_map" in feature_extractor_dict and not is_local:
-            feature_extractor_dict["auto_map"] = add_model_info_to_auto_map(
-                feature_extractor_dict["auto_map"], pretrained_model_name_or_path
-            )
+        if not is_local:
```

Comment:
just curious as to why we only do it if not local whereas before we did it for local and not local? 🤗

Reply:
there is a nested if statements in there 
-if
--- if
--- if

Reply:
> what does `is_local` mean ? 

if i call a model using this 
```python
path = "./folder1/folder2/folder3"
from transformers import AutoModelForImageClassification
model = AutoModelForImageClassification.from_pretrained(path, trust_remote_code=True)
```
if the path exists in my pc then we load from there 
> what if the `is_local` is false ? 

then in this case we call the model from the hub and add the tag
> is the tag added when we are only calling a custom model from huggingface ?

yes, there is no need to add the tag if we are calling the model from the local pc

hope this answers your questions 🤗